.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  - push
+  - pull_request
+name: ci
+jobs:
+  lint:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: install node v10
+        uses: actions/setup-node@v1
+        with:
+          node-version: v10.x
+      - name: install dependencies
+        run: npm install
+      - name: check lint
+        run: make check-lint
+  test:
+    name: test
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - ubuntu-latest
```

Comment:
Should we also test on OS X?

Reply:
tests are super-flaky on `macos-latest`. The tests for request closed are failing randomly... I will leave `macos` out until we have time to review them.